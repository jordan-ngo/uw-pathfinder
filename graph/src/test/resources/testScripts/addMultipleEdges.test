#Create a graph
CreateGraph graph

#Add three nodes to the graph
AddNode graph one
AddNode graph two
AddNode graph three

#Add edges to the graph from one to two, two to three, and three to one with labels
AddEdge graph one two sugar
AddEdge graph two three spice
AddEdge graph three one nice

#Output the list of all nodes in the graph
ListNodes graph

#Output the list of children for node one, two, and three
ListChildren graph one
ListChildren graph two
ListChildren graph three