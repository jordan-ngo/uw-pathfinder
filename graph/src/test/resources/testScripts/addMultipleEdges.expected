#Create a graph
created graph graph

#Add three nodes to the graph
added node one to graph
added node two to graph
added node three to graph

#Add edges to the graph from one to two, two to three, and three to one with labels
added edge sugar from one to two in graph
added edge spice from two to three in graph
added edge nice from three to one in graph

#Output the list of all nodes in the graph
graph contains: one three two

#Output the list of children for node one, two, and three
the children of one in graph are: two(sugar)
the children of two in graph are: three(spice)
the children of three in graph are: one(nice)


